### DISCUSSION TITLE
Creating reciprocal linkages to BOLD

### DISCUSSION THREAD

**Discussion by campmlc** [ArctosDB/arctos#8642] [General]
Email sent 2-2-2020

Live links to voucher specimens from BOLD
Mariel Lee Campbell <campmlc@unm.edu>
Sun 2/2/2020 4:41 PM
To: support@boldsystems.org <support@boldsystems.org>
Hello,
I am writing to see if it is possible for BOLD to include/allow direct hot links from a BOLD record to a museum specimen record. I represent over 200 natural history and cultural collections who are using the Arctos collection management system, an online platform that provides each specimen record with a linkable url. For example, see https://arctos.database.museum/guid/MSB:Para:21002

If specimens were deposited correctly by the collector/sequence submitter, the specimen record should have a live link out from our database to the BOLD record, and hopefully at BOLD the record will contain the three-part guid (institution code: collection code: catalog number) that points back to the specimen identifier in our system, as above. However, we would like to explore having this guid identifier converted to the live url linkage so that a viewer could click the museum ID and land immediately at the specimen record page of the museum voucher.  We currently have the ability to create these kinds of reciprocal live linkages with GenBank. Would this be possible at BOLD? For example:
UAMIC1999-14 and http://arctos.database.museum/guid/UAM:Ento:142554

On a similar line, there are also records at BOLD that reference museum specimens, but the submitter of the record did not use the museum catalog number, instead submitting a field number or other ID. However, we can track back to the museum catalog number/guid in our database.
For example, see:
 BTVC131-05 and https://arctos.database.museum/guid/MSB:Mamm:155929

and NEON example (our members include NEON external repositories):
MUP008-14 and http://arctos.database.museum/guid/MSB:Mamm:299226

Does BOLD allow external parties to append information to a record? In this case, the institution housing the specimen in perpetuity wishes to make proper attribution to the specimen voucher by linking to a museum catalog number.

The ability to link genetic sequence information back to a museum voucher specimen is essential not only for morphological verification and in the interests of replicability and scientific accuracy, but also allows linkage to additional research that has been conducted using that specimen. We are very interested in tracking down and linking all usage for specimens archived in our institutions for integration and attribution purposes and to add value to the specimen data across all repositories. Would it be possible for us to initiate a conversation with BOLD as to how to increase collaboration and data linkage between our repositories?

Thank you for considering our request. I look forward to hearing from you.

Mariel Campbell
Arctos Working Group Co-Chair
Collection Manager, Division of Genomic Resources
Museum of Southwestern  Biology

**Comment by DerekSikes** [comment]
BOLD has modified their data upload form to allow external URLs - some
years ago. HOWEVER, these are only visible 'behind the scenes' to those
with login & editing access & projects in BOLD. The public records/database
lack these link-backs.

I hope your email leads to some progress. I'd be happy to follow up. It's
silly they've been telling me for years that this is 'in the works'...

-Derek

On Sun, Feb 2, 2020 at 2:48 PM Mariel Campbell <notifications@github.com>
wrote:

> Assigned #2482 <https://github.com/ArctosDB/arctos/issues/2482> to
> @DerekSikes <https://github.com/DerekSikes>.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ArctosDB/arctos/issues/2482?email_source=notifications&email_token=ACFNUMZRA5JEIQGACIXUE6LRA5LUDA5CNFSM4KO44VCKYY3PNVWWK3TUL52HS4DFWZEXG43VMVCXMZLOORHG65DJMZUWGYLUNFXW5KTDN5WW2ZLOORPWSZGOWLLH2RY#event-3000401223>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACFNUM25B2IUUF7VEKXCG7LRA5LUDANCNFSM4KO44VCA>
> .
>


-- 

+++++++++++++++++++++++++++++++++++
Derek S. Sikes, Curator of Insects
Professor of Entomology
University of Alaska Museum
1962 Yukon Drive
Fairbanks, AK   99775-6960

dssikes@alaska.edu

phone: 907-474-6278
FAX: 907-474-5469

University of Alaska Museum  -  search 400,276 digitized arthropod records
http://arctos.database.museum/uam_ento_all
<http://www.uaf.edu/museum/collections/ento/>
+++++++++++++++++++++++++++++++++++

Interested in Alaskan Entomology? Join the Alaska Entomological
Society and / or sign up for the email listserv "Alaska Entomological
Network" at
http://www.akentsoc.org/contact_us

**Comment by mkoo** [comment]
@campmlc Did they followup?
Pls update or close!

**Comment by DerekSikes** [comment]
I just sent another email to BOLD. Will update with anything I learn.

-Derek


On Thu, Feb 3, 2022 at 2:23 PM Michelle Koo ***@***.***>
wrote:

> @campmlc <https://github.com/campmlc> Did they followup?
> Pls update or close!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ArctosDB/arctos/issues/2482#issuecomment-1029494516>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACFNUMYR5TZSFFFQMNWKZPLUZME6ZANCNFSM4KO44VCA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

+++++++++++++++++++++++++++++++++++
*Derek S. Sikes*, Curator of Insects, Professor of Entomology
University of Alaska Museum (UAM), University of Alaska Fairbanks
1962 Yukon Drive, Fairbanks, AK   99775-6960
***@***.*** phone: 907-474-6278 he/him/his
University of Alaska Museum <https://www.uaf.edu/museum/collections/ento/>
-  search 357,704 digitized arthropod records
<http://arctos.database.museum/uam_ento>
+++++++++++++++++++++++++++++++++++

Interested in Alaskan Entomology? Join the Alaska Entomological
Society and / or sign up for the email listserv "Alaska Entomological
Network" at
http://www.akentsoc.org/contact_us

**Comment by dustymc** [comment]
Status?

**Comment by DerekSikes** [comment]
No update. I've emailed support & cc'd Paul Hebert, chief honcho, this time.

-Derek

On Mon, Aug 1, 2022 at 10:19 AM dustymc ***@***.***> wrote:

> Status?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ArctosDB/arctos/issues/2482#issuecomment-1201549727>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACFNUM2F22Y4B22WYHJSQALVXAIMZANCNFSM4KO44VCA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

+++++++++++++++++++++++++++++++++++
*Derek S. Sikes*, Curator of Insects, Professor of Entomology
University of Alaska Museum (UAM), University of Alaska Fairbanks
1962 Yukon Drive, Fairbanks, AK   99775-6960
***@***.*** phone: 907-474-6278 he/him/his
University of Alaska Museum <https://www.uaf.edu/museum/collections/ento/>
-  search 357,704 digitized arthropod records
<http://arctos.database.museum/uam_ento>
+++++++++++++++++++++++++++++++++++

Interested in Alaskan Entomology? Join the Alaska Entomological
Society and / or sign up for the email listserv "Alaska Entomological
Network" at
http://www.akentsoc.org/contact_us

**Comment by Jegelewicz** [comment]
@campmlc @DerekSikes it seems like we aren't getting anywhere? Does this need to remain open?

**Comment by DerekSikes** [comment]
Shouldn't it remain open until it's resolved? Someday they'll add these links and presumably the more we pester them the sooner that day will be?

**Comment by campmlc** [comment]
Agree. We just need to pester them more. Incredible that they haven't figured this out yet.

**Comment by Jegelewicz** [comment]
Get to pestering? I am just trying to clean up old things and this is one of the oldest....

**Comment by DerekSikes** [comment]
just sent this:
	
to Sujeevan, Paul
Paul & Sujeevan,

Hoping that 2023 will be the year these public reciprocal linkages can be made available in BOLD (like they are in GenBank). All the records I upload from our database have URLs links back to our database records - and we publicly link to BOLD records, so it would be great if those URLs could already present in BOLD could be made public so users can link to the original specimen records.

Thanks!
Derek

**Comment by campmlc** [comment]
Any response? BoLD has NEON records identified as "Chordata" that have vouchers identified to species level at MSB - might be helpful to them to consider linking to our database . . .

**Comment by DerekSikes** [comment]
No response from them. If I run into them in person I'll bring it up but I think we can call this a dead end for now.

**Comment by mkoo** [comment]
Closed due to inaction from 3rd party

**Comment by DerekSikes** [comment]
Looks like the new BOLD v5 has these, sort of... check out: https://bench.boldsystems.org/index.php/Public_RecordView?processid=UAMIC4896-23

and scroll down to SPECIMEN DETAILS and you will see this URL: https://arctos.database.museum/guid/UAM:Ento:472972

BUT... it's not a live link, just text that someone would have to copy& paste to use.
